LICENSE.APL.txt

```diff
@@ -0,0 +1,37 @@
+==========================================================
+Licensing Terms for he PyInstaller community runtime hooks
+==========================================================
+
+The PyInstaller community **runtime** hooks are licensed under the terms of the
```

Comment:
Again: Maybe add something like "these reside in rthooks/"?

Reply:
Done.